### PR TITLE
[New Rule] Alternate Data Stream Creation at Volume Root Directory

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Identifies the creation of an Alternate Data Stream (ADS) at the volume root directory (C:), which can indicate the attempt to hide tools and malware, as ADSs created in this directory are not displayed by system utilities.

## Sample Data

<details>
<summary> Sample Data </summary>

```
{
  "_index": ".ds-logs-endpoint.events.file-default-2024.03.14-000021",
  "_id": "whouP44B1-PCmUgAcky4",
  "_version": 1,
  "_score": 0,
  "_source": {
    "agent": {
      "id": "418f0319-3919-406c-ae6e-af5b3514ef50",
      "type": "endpoint",
      "version": "8.12.2"
    },
    "process": {
      "Ext": {
        "ancestry": [
          "NDE4ZjAzMTktMzkxOS00MDZjLWFlNmUtYWY1YjM1MTRlZjU3LTQwODgtMTcxMDQ1NjAwMC4xNDQyMTM3MDA=",
          "NDE4ZjAzMTktMzkxOS00MDZjLWFlNmUtYWY1YjM1MTRlZjU3LTIwMDgtMTcxMDQ1MDI1MS4xOTEyNTk4MDA="
        ],
        "code_signature": [
          {
            "trusted": true,
            "subject_name": "Microsoft Windows",
            "exists": true,
            "status": "trusted"
          }
        ],
        "hidden": {
          "behaviors": {
            "network": {}
          }
        }
      },
      "parent": {
        "pid": 4088
      },
      "code_signature": {
        "trusted": true,
        "subject_name": "Microsoft Windows",
        "exists": true,
        "status": "trusted"
      },
      "name": "powershell.exe",
      "pid": 7452,
      "thread": {
        "Ext": {
          "call_stack_summary": "ntdll.dll|kernelbase.dll|system.management.automation.ni.dll|microsoft.powershell.commands.management.ni.dll|system.management.automation.ni.dll|mscorlib.ni.dll|system.management.automation.ni.dll|microsoft.powershell.commands.management.ni.dll|system.management.automation.ni.dll|Unbacked",
          "call_stack_contains_unbacked": true,
          "call_stack": [
            {
              "symbol_info": "C:\\Windows\\System32\\ntdll.dll!ZwCreateFile+0x14"
            },
            {
              "symbol_info": "C:\\Windows\\System32\\KernelBase.dll!CreateFileW+0x5f9"
            },
            {
              "symbol_info": "C:\\Windows\\System32\\KernelBase.dll!CreateFileW+0x66"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x10bbc3e"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x135d777"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x12326cf"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x11198b0"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x11197bc"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\Microsoft.Pae3498d9#\\b6086e57764707b440556afa37e9c9ff\\Microsoft.PowerShell.Commands.Management.ni.dll+0x14d05e"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\Microsoft.Pae3498d9#\\b6086e57764707b440556afa37e9c9ff\\Microsoft.PowerShell.Commands.Management.ni.dll+0x14da16"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x1181443"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x10d9a2b"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x115ebe4"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\mscorlib\\316b66ba76e2cd70541b5fe8161637c9\\mscorlib.ni.dll+0xe31b21"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x114f9c5"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x11f41fb"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\Microsoft.Pae3498d9#\\b6086e57764707b440556afa37e9c9ff\\Microsoft.PowerShell.Commands.Management.ni.dll+0x1417e6"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x1181443"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x10d9a2b"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x116404c"
            },
            {
              "symbol_info": "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x12675af"
            },
            {
              "symbol_info": "Unbacked+0x37f1",
              "callsite_trailing_bytes": "834648fab8010000004883c4685b5d5e5f415c415d415e415fc3e8d02aac5fcc0000001910090010c20c300b500a60097008c006d004e002f000004000000002",
              "protection": "RWX",
              "callsite_leading_bytes": "2428f6c1017404488b49ffe8bf43735f4889442430498bd5448b44245c4c8b4c2448488b6c244048896c2420488b6c243848896c2428488d4b08488b09ff5318"
            }
          ]
        },
        "id": 13612
      },
      "entity_id": "NDE4ZjAzMTktMzkxOS00MDZjLWFlNmUtYWY1YjM1MTRlZjU3LTc0NTItMTcxMDQ1NzAxNi45OTk3MDI5MDA=",
      "executable": "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
    },
    "@timestamp": "2024-03-14T22:56:57.2682253Z",
    "file": {
      "Ext": {
        "header_data": [],
        "entropy": 4.132760306568156,
        "header_bytes": "4d5a90000300000004000000ffff0000",
        "windows": {
          "zone_identifier": -1
        },
        "monotonic_id": 131
      },
      "path": "C:\\:Host Process for Windows Service",
      "size": 289794,
      "name": ":Host Process for Windows Service"
    },
    "ecs": {
      "version": "8.10.0"
    },
    "data_stream": {
      "namespace": "default",
      "type": "logs",
      "dataset": "endpoint.events.file"
    },
    "elastic": {
      "agent": {
        "id": "418f0319-3919-406c-ae6e-af5b3514ef50"
      }
    },
    "host": {
      "hostname": "desktop",
      "os": {
        "Ext": {
          "variant": "Windows 10 Enterprise Evaluation"
        },
        "kernel": "22H2 (10.0.19045.3930)",
        "name": "Windows",
        "family": "windows",
        "type": "windows",
        "version": "22H2 (10.0.19045.3930)",
        "platform": "windows",
        "full": "Windows 10 Enterprise Evaluation 22H2 (10.0.19045.3930)"
      },
      "ip": [
        "127.0.0.1",
        "::1"
      ],
      "name": "desktop",
      "id": "9f87f8dd-e42d-4d6d-bd6a-031dcf50bc27",
      "mac": [
        "00-0c-29-ec-d0-f0"
      ],
      "architecture": "x86_64"
    },
    "event": {
      "agent_id_status": "verified",
      "sequence": 4501,
      "ingested": "2024-03-14T22:56:32Z",
      "created": "2024-03-14T22:56:57.2682253Z",
      "kind": "event",
      "module": "endpoint",
      "action": "creation",
      "id": "NTBxPr286/g68BVQ+++++3jL",
      "category": [
        "file"
      ],
      "type": [
        "creation"
      ],
      "dataset": "endpoint.events.file",
      "outcome": "unknown"
    },
    "message": "Endpoint file event",
    "user": {
      "domain": "DESKTOP",
      "name": "Jonhnathan",
      "id": "S-1-5-21-3897597994-1868010319-314229934-1001"
    }
  },
  "fields": {
    "file.path": [
      "C:\\:Host Process for Windows Service"
    ],
    "host.os.full.text": [
      "Windows 10 Enterprise Evaluation 22H2 (10.0.19045.3930)"
    ],
    "event.category": [
      "file"
    ],
    "process.name.text": [
      "powershell.exe"
    ],
    "host.os.name.text": [
      "Windows"
    ],
    "host.os.full": [
      "Windows 10 Enterprise Evaluation 22H2 (10.0.19045.3930)"
    ],
    "process.parent.pid": [
      4088
    ],
    "host.hostname": [
      "desktop-b2hbsh1"
    ],
    "process.pid": [
      7452
    ],
    "host.mac": [
      "00-0c-29-ec-d0-f0",
      "8c-1d-96-ad-81-57"
    ],
    "process.code_signature.exists": [
      true
    ],
    "elastic.agent.id": [
      "418f0319-3919-406c-ae6e-af5b3514ef57"
    ],
    "process.thread.Ext.call_stack_contains_unbacked": [
      true
    ],
    "process.code_signature.subject_name": [
      "Microsoft Windows"
    ],
    "host.os.version": [
      "22H2 (10.0.19045.3930)"
    ],
    "process.thread.Ext.call_stack.symbol_info": [
      "C:\\Windows\\System32\\ntdll.dll!ZwCreateFile+0x14",
      "C:\\Windows\\System32\\KernelBase.dll!CreateFileW+0x5f9",
      "C:\\Windows\\System32\\KernelBase.dll!CreateFileW+0x66",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x10bbc3e",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x135d777",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x12326cf",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x11198b0",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x11197bc",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\Microsoft.Pae3498d9#\\b6086e57764707b440556afa37e9c9ff\\Microsoft.PowerShell.Commands.Management.ni.dll+0x14d05e",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\Microsoft.Pae3498d9#\\b6086e57764707b440556afa37e9c9ff\\Microsoft.PowerShell.Commands.Management.ni.dll+0x14da16",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x1181443",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x10d9a2b",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x115ebe4",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\mscorlib\\316b66ba76e2cd70541b5fe8161637c9\\mscorlib.ni.dll+0xe31b21",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x114f9c5",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x11f41fb",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\Microsoft.Pae3498d9#\\b6086e57764707b440556afa37e9c9ff\\Microsoft.PowerShell.Commands.Management.ni.dll+0x1417e6",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x1181443",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x10d9a2b",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x116404c",
      "C:\\Windows\\assembly\\NativeImages_v4.0.30319_64\\System.Manaa57fc8cc#\\83f2abc4d409a28d6f0ab07c44c06343\\System.Management.Automation.ni.dll+0x12675af",
      "Unbacked+0x37f1"
    ],
    "host.os.name": [
      "Windows"
    ],
    "host.name": [
      "desktop-b2hbsh1"
    ],
    "event.agent_id_status": [
      "verified"
    ],
    "event.kind": [
      "event"
    ],
    "event.outcome": [
      "unknown"
    ],
    "process.code_signature.trusted": [
      true
    ],
    "file.path.text": [
      "C:\\:Host Process for Windows Service"
    ],
    "host.os.type": [
      "windows"
    ],
    "user.id": [
      "S-1-5-21-3897597994-1868010319-314229934-1001"
    ],
    "process.Ext.ancestry": [
      "NDE4ZjAzMTktMzkxOS00MDZjLWFlNmUtYWY1YjM1MTRlZjU3LTQwODgtMTcxMDQ1NjAwMC4xNDQyMTM3MDA=",
      "NDE4ZjAzMTktMzkxOS00MDZjLWFlNmUtYWY1YjM1MTRlZjU3LTIwMDgtMTcxMDQ1MDI1MS4xOTEyNTk4MDA="
    ],
    "data_stream.type": [
      "logs"
    ],
    "host.architecture": [
      "x86_64"
    ],
    "file.Ext.header_bytes": [
      "4d5a90000300000004000000ffff0000"
    ],
    "process.name": [
      "powershell.exe"
    ],
    "agent.id": [
      "418f0319-3919-406c-ae6e-af5b3513ef57"
    ],
    "ecs.version": [
      "8.10.0"
    ],
    "event.created": [
      "2024-03-14T22:56:57.268Z"
    ],
    "agent.version": [
      "8.12.2"
    ],
    "host.os.family": [
      "windows"
    ],
    "process.thread.id": [
      13612
    ],
    "process.thread.Ext.call_stack_summary": [
      "ntdll.dll|kernelbase.dll|system.management.automation.ni.dll|microsoft.powershell.commands.management.ni.dll|system.management.automation.ni.dll|mscorlib.ni.dll|system.management.automation.ni.dll|microsoft.powershell.commands.management.ni.dll|system.management.automation.ni.dll|Unbacked"
    ],
    "file.Ext.monotonic_id": [
      131
    ],
    "file.Ext.windows.zone_identifier": [
      "-1"
    ],
    "user.name": [
      "Jonhnathan"
    ],
    "process.entity_id": [
      "NDE4ZjAzMTktMzkxOS00MDZjLWFlNmUtYWY1YjM1MTRlZjU3LTc0NTItMTcxMDQ1NzAxNi45OTk3MDI5MDA="
    ],
    "process.thread.Ext.call_stack.callsite_trailing_bytes": [
      "834648fab8010000004883c4685b5d5e5f415c415d415e415fc3e8d02aac5fcc0000001910090010c20c300b500a60097008c006d004e002f000004000000002"
    ],
    "process.thread.Ext.call_stack.protection": [
      "RWX"
    ],
    "event.sequence": [
      4501
    ],
    "host.ip": [
      "127.0.0.1",
      "::1"
    ],
    "process.executable.caseless": [
      "c:\\windows\\system32\\windowspowershell\\v1.0\\powershell.exe"
    ],
    "agent.type": [
      "endpoint"
    ],
    "process.executable.text": [
      "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
    ],
    "event.module": [
      "endpoint"
    ],
    "host.os.kernel": [
      "22H2 (10.0.19045.3930)"
    ],
    "host.os.full.caseless": [
      "windows 10 enterprise evaluation 22h2 (10.0.19045.3930)"
    ],
    "user.domain": [
      "DESKTOP-B2HBSH1"
    ],
    "host.id": [
      "9f87f8dd-e62d-4d6d-bd6a-031dcf50bc27"
    ],
    "process.name.caseless": [
      "powershell.exe"
    ],
    "process.executable": [
      "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
    ],
    "file.Ext.entropy": [
      4.132760306568156
    ],
    "data_stream.namespace": [
      "default"
    ],
    "file.name": [
      ":Host Process for Windows Service"
    ],
    "file.size": [
      289794
    ],
    "process.code_signature.status": [
      "trusted"
    ],
    "message": [
      "Endpoint file event"
    ],
    "host.os.Ext.variant": [
      "Windows 10 Enterprise Evaluation"
    ],
    "process.thread.Ext.call_stack.callsite_leading_bytes": [
      "2428f6c1017404488b49ffe8bf43735f4889442430498bd5448b44245c4c8b4c2448488b6c244048896c2420488b6c243848896c2428488d4b08488b09ff5318"
    ],
    "event.action": [
      "creation"
    ],
    "event.ingested": [
      "2024-03-14T22:56:32.000Z"
    ],
    "@timestamp": [
      "2024-03-14T22:56:57.268Z"
    ],
    "host.os.platform": [
      "windows"
    ],
    "file.path.caseless": [
      "c:\\:host process for windows service"
    ],
    "data_stream.dataset": [
      "endpoint.events.file"
    ],
    "event.type": [
      "creation"
    ],
    "process.Ext.code_signature": [
      {
        "trusted": [
          true
        ],
        "subject_name": [
          "Microsoft Windows"
        ],
        "exists": [
          true
        ],
        "status": [
          "trusted"
        ]
      }
    ],
    "event.id": [
      "NTBxPr286/g68BVQ+++++3jL"
    ],
    "event.dataset": [
      "endpoint.events.file"
    ],
    "host.os.name.caseless": [
      "windows"
    ],
    "user.name.text": [
      "Jonhnathan"
    ]
  }
}
```

</details>

### CODE REVIEW

File: rules/windows/defense_evasion_root_dir_ads_creation.toml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+updated_date = "2024/03/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of an Alternate Data Stream (ADS) at the volume root directory (C:), which can indicate the
+attempt to hide tools and malware, as ADSs created in this directory are not displayed by system utilities.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.sysmon_operational-*"]
```
Comment (brokensound77) [inline rules/windows/defense_evasion_root_dir_ads_creation.toml:16]:
```suggestion
index = ["winlogbeat-*", "logs-endpoint.events.file-*", "logs-windows.sysmon_operational-*"]
```

File: rules/windows/defense_evasion_root_dir_ads_creation.toml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+risk_score = 47
+rule_id = "ff6cf8b9-b76c-4cc1-ac1b-4935164d1029"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Data Source: Elastic Defend", "Data Source: Sysmon"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where host.os.type == "windows" and event.type == "creation" and
+  startsWith(file.path, "C:\\:")
```
Comment (brokensound77) [inline rules/windows/defense_evasion_root_dir_ads_creation.toml:30]: question - should we make this a bit wider to catch _any_ root dir?

could keep it efficient with regex

```suggestion
  file.path regex~ """[a-zA-Z]:\\:.+"""
```

May need to test that

Comment (brokensound77) [reply rules/windows/defense_evasion_root_dir_ads_creation.toml:30]: if so, update description

Comment (Samirbous) [inline rules/windows/defense_evasion_root_dir_ads_creation.toml:30]: to catch both file or process execution, `regex~` is already case insensitive.


```suggestion
any where host.os.type == "windows" and event.category in ("file", "process") and 
 (
   (event.type == "creation" and file.path regex~ """[A-Z]:\\:.+""") or 
   
   (event.type == "start" and process.executable regex~ """[A-Z]:\\:.+""")
   )
```

File: rules/windows/defense_evasion_root_dir_ads_creation.toml (hunk 1 above)
Comment (Samirbous) [reply rules/windows/defense_evasion_root_dir_ads_creation.toml:16]: if you agree with suggestion to query both file and process events then maybe extend it to the system index

**Comment by terrancedejesus** [comment]
Great addition! Couple of questions:

* I saw that a persistent Windows service was created for establishing a reverse SSH tunnel where the binary location was in the volume root directory. Does it make sense to add another rule that covers this as well? I'd assume LOLBins can be leveraged to some degree with ADS and root directories.
* Do we have any idea if this rule would be noisy? Maybe depending on environment of course. If so, I'd be curious if we can focus on `file.size` here to identify anomalous/malicious executables?

File: rules/windows/defense_evasion_root_dir_ads_creation.toml (hunk 1 above)
Comment (w0rk3r) [reply rules/windows/defense_evasion_root_dir_ads_creation.toml:16]: Added both indexes.

**Comment by w0rk3r** [comment]
> I saw that a persistent Windows service was created for establishing a reverse SSH tunnel where the binary location was in the volume root directory. Does it make sense to add another rule that covers this as well? I'd assume LOLBins can be leveraged to some degree with ADS and root directories as the executables can be called directly or invoked via PS.

The changes proposed by samir will cover process execution too

> Do we have any idea if this rule would be noisy?

I don't think so

**Review by Aegrah — APPROVED** [review]

**Review by Samirbous — APPROVED** [review]

**Pull request merged**